dolphin/db/sqlalchemy/api.py

```diff
@@ -24,15 +24,15 @@
 from oslo_db import options as db_options
 from oslo_db.sqlalchemy import session
 from oslo_log import log
-from sqlalchemy import  create_engine
+from sqlalchemy import create_engine
 from dolphin.db.sqlalchemy import models
 from dolphin.db.sqlalchemy.models import Storage, RegistryContext
 
 CONF = cfg.CONF
 LOG = log.getLogger(__name__)
 _FACADE = None
 
-_DEFAULT_SQL_CONNECTION = 'sqlite:///'
+_DEFAULT_SQL_CONNECTION = 'sqlite:////var/lib/dolphin'
```

Comment:
Can it be configurable?

Reply:
read from conf

Reply:
yes we red  form config only.. removed path in default_connect

Reply:
Ok